dist/server/index.js

```diff
@@ -0,0 +1,68 @@
+//////////////////////////////////////////////////
+// Silex, live web creation
+// http://projects.silexlabs.org/?/silex/
+//
+// Copyright (c) 2012 Silex Labs
+// http://www.silexlabs.org/
+//
+// Silex is available under the GPL license
+// http://www.silexlabs.org/silex/silex-licensing/
+//////////////////////////////////////////////////
+
+'use strict';
+
+// node modules
+const Path = require('path');
+const fs = require('fs');
+const express = require('express');
+const compression = require('compression');
+const cookieParser = require('cookie-parser');
+const session = require('cookie-session');
+const serveStatic = require('serve-static');
+const WebsiteRouter = require('./WebsiteRouter.js');
+const CloudExplorerRouter = require('./CloudExplorerRouter.js');
+const PublishRouter = require('./PublishRouter.js');
+const SslRouter = require('./SslRouter.js');
+
+// 6805 is the date of sexual revolution started in paris france 8-)
+const port = process.env.PORT || 6805;
+const rootUrl = process.env.SERVER_URL || `http://localhost:${port}`;
+
+const app = express();
+
+// compress gzip when possible
+app.use(compression());
+
+// cookie & session
+app.use(cookieParser());
+app.use(session({
+  name: 'silex-session',
+  secret: process.env.SILEX_SESSION_SECRET || 'test session secret'
+}));
+
+// create the routes for unifile/CloudExplorer
+// and for Silex tasks
+const cloudExplorerRouter = new CloudExplorerRouter(rootUrl);
+app.use(new WebsiteRouter(port, rootUrl, cloudExplorerRouter.unifile));
+app.use(new PublishRouter(port, rootUrl, cloudExplorerRouter.unifile));
+app.use(new SslRouter(app));
```

Comment:
I know it's arguable, but IMO this file should act as a controller and perform the environment checks (like SSL and such) before using the router.
In that case, the router wouldn't have to worry about the env and the configuration variables should be injected in its constructor. 

This will greatly simplify testing.

Reply:
:+1: 
I'll try to refactor
If you could write this in a separate issue that would be great - not a problem otherwise, I'll try